src/themes/motoScout24.ts

```diff
@@ -13,6 +13,7 @@ export const motoScout24Config = mergeConfigs(
       semanticTokens: {
         colors: {
           brand: {
+            fg: { value: '{colors.white}' },
```

Comment:
Semantic tokens to handle the case mentioned [here](https://github.com/smg-automotive/components-pkg/pull/1206/commits/99db3c058fd9270117471610b11a37326441f98b#r1848613203)